es-vue-base/package.json

```diff
@@ -38,15 +44,31 @@
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^28.1.3",
     "cross-env": "^7.0.3",
+    "eslint": "^8.20.0",
+    "eslint-config-energysage": "^2.2.1",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-import-resolver-webpack": "^0.13.2",
```

Comment:
Should https://www.npmjs.com/package/@rollup/plugin-eslint be used in this project?

Reply:
I didn't really understand this plugins use case. It seems to just look at the rollup config for files to lint, given we are linting everything I didn't think it was necessary.